docs/getting-started-guides/vagrant.md

```diff
@@ -1,14 +1,15 @@
 ## Getting started with Vagrant
 
+Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/develop on your local machine (Linux, Mac OS X).
+
 ### Prerequisites
 1. Install latest version >= 1.6.2 of vagrant from http://www.vagrantup.com/downloads.html
 2. Install latest version of Virtual Box from https://www.virtualbox.org/wiki/Downloads
-3. Install the `net-tools` package for your distribution for VirtualBox's private networks.
 4. Get or build a [binary release](binary_release.md)
 
 ### Setup
 
-By default, the Vagrant setup will create a single kubernetes-master and 1 kubernetes-minion. Each VM will take 512 MB, so make sure you have at least 2 GB of free memory. To start your local cluster, open a shell and run:
+By default, the Vagrant setup will create a single kubernetes-master and 1 kubernetes-minion. Each VM will take 512 MB, so make sure you have at least 2 GB to 4GB of free memory (plus appropriate free disk space). To start your local cluster, open a shell and run:
```

Comment:
The default VM memory usage has been bumped up to 1024 MB in order to allow for more room in /tmpfs during Salt provisioning.